src/NServiceBus.Core/App_Packages/Particular.Licensing/FindActiveLicense/LicenseSourceHKCURegKey.cs

```diff
@@ -43,21 +40,25 @@ string ReadLicenseFromRegistry(RegistryView view)
                 {
                     if (registryKey == null)
                     {
-                        return null;
+                        return ValidateLicense(null, applicationName);
```

Comment:
have you verified this giving the expected output? Following the code path (which is quite long for passing some null parameter...) it looks like this would end up with a `License found in {location} is not valid - System.Exception: Empty license string` message.

not sure I like the approach of that null value being passed into ~4 methods before it throws an exception, which is then catched again.